packages/grafana-data/src/transformations/transformers/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const transformationsVariableSupport = () => {
+  return (window as any)?.grafanaBootData?.settings?.featureToggles?.transformationsVariableSupport;
+};
```

Comment:
@oscarkilhed could you do
```
import { BootData } from "../../types";

declare global {
  interface Window {
    grafanaBootData: BootData;
  }
};
export const transformationsVariableSupport = () => {
  return window.grafanaBootData.settings.featureToggles.transformationsVariableSupport;
};
}
```
instead?

Reply:
I'll try! The way I did it was used elsewhere as well. I'll try to create a more generic function for it, and replace all uses. 

The grafana boot data leaking into grafana-data is not great :(